lisa/tools/stat.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+
+class Stat(Tool):
+    @property
+    def command(self) -> str:
+        return "stat"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    def install(self) -> bool:
+        posix_os: Posix = cast(Posix, self.node.os)
+        package_name = "coreutils"
+        posix_os.install_packages(package_name)
+        return self._check_exists()
+
+    def get_block_size_fs(self, file: str) -> int:
```

Comment:
What's the last "fs" for? Is it "file system", but the input argument is "file"? Anyway the name should be like `get_fs_block_size`.

Reply:
file system

Reply:
update